### PR TITLE
Create  Message-Encrypter-Decrypter

### PR DESCRIPTION
**Pull request by eunicek-333**

### CODE REVIEW

File: _data/projects/Message-Encrypter-Decrypter [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (shiftkey) [inline _data/projects/Message-Encrypter-Decrypter:1]: This file needs to have a `.yml` extension for it to be correctly included in the project

Comment (ritwik12) [reply _data/projects/Message-Encrypter-Decrypter:1]: @eunicek-333 Ping!!

File: _data/projects/Message-Encrypter-Decrypter [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+name: Message-Encrypter-Decrypter
+desc: Encryption is the process by which the original message(plaintext) is converted to an unreadable(ciphertext) form to prevent unauthorized parties from reading it.
```
Comment (ritwik12) [inline _data/projects/Message-Encrypter-Decrypter:3]: @eunicek-333 Please try to make the Description a bit shorter for the ease.

File: _data/projects/Message-Encrypter-Decrypter [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+site: https://github.com/eunicek-333/Message-Encrypter-Decrypter
+tags:
+- python
+- pip
+- tkinter
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/eunicek-333/Message-Encrypter-Decrypter/labels/up-for-grabs
+stats:
+  issue-count: 0
```
Comment (ritwik12) [inline _data/projects/Message-Encrypter-Decrypter:15]: @eunicek-333 You don't need to add these. It will be added by our day-to-day Project Stats PR.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by shiftkey** [comment]
Closing this out due to inactivity.

@eunicek-333 if you still wish to see this project listed, please look over the feedback and submit a fresh pull request.

**Pull request closed without merging**